registry/mavrickrishi/modules/nexus/README.md

```diff
@@ -0,0 +1,132 @@
+---
+display_name: Sonatype Nexus Repository
+description: Configure package managers to use Sonatype Nexus Repository for Maven, npm, PyPI, and Docker registries.
+icon: /.icons/nexus.svg
```

Comment:
This is not the correct path. It should be the relative path to .icons/nexus.svg